addons/survey/tests/common.py

```diff
@@ -314,3 +314,80 @@ def setUp(self):
             'sequence': 3,
             'question_type': 'numerical_box',
         })
+
+
+class TestSurveyResultsCommon(SurveyCase):
+    def setUp(self):
+        super(TestSurveyResultsCommon, self).setUp()
+        self.survey_manager = mail_new_test_user(
+            self.env, name='Gustave Doré', login='survey_manager', email='survey.manager@example.com',
+            groups='survey.group_survey_manager,base.group_user'
+        )
+
+        # Create survey with questions
+        self.survey = self.env['survey.survey'].create({
+            'title': 'Test Survey Results',
+            'questions_layout': 'one_page'
+        })
+        self.question_cb = self._add_question(
```

Comment:
"question_char_box", "question_numerical", ...

It's slightly longer but it makes it more readable.